docs/_documentations/overview.md

```diff
@@ -9,21 +9,72 @@ type: document
 
-Write code, track application and build statuses, view project logs, and run your application. Codewind supports development of Appsody, Go, Java Lagom, Microprofile/Java EE, Node.js, Python, Spring, and Swift containerized projects. In addition, the tools support easily debugging Microprofile/Java EE, Spring, and Node.js applications.
+When it comes to running your microservices on the cloud, Codewind enables you to build and run Docker-containerised applications on [Kubernetes](https://kubernetes.io/) container orchestration applications, including [Openshift](https://www.openshift.com/), without the need for in-depth knowledge of container technology.
 
+## How does Codewind help me develop containerised applications?
+Codewind improves your inner loop experience, enabling you to create a microservice quickly, rapidly iterate on changes and make improvements to performance thanks to the following features:
 
-# Features
+1. **Access to pre-configured containerised project templates** Codewind provides pre-configured, Docker-containerised project templates ADD URL covering several languages including [Node.js](https://nodejs.dev/), Java, Python and [Swift](https://swift.org/), and several frameworks such as Express, Spring Boot and Open Liberty ADD URLS, with more on the way. [Quickly create and deploy microservices in languages and frameworks that you're familiar with](https://www.youtube.com/watch?v=zKMggp10gq4&t=12s). Modify the pre-configured projects to develop your customised microservice.
+
+2. **Ability to add your own applications** Access the seamless container development and deployment experience on your IDE that Codewind offers for your current projects.
+
+3. **Auto-build-and-run on code changes** As you develop, Codewind *automatically* pushes your code changes to your container as efficiently as possible. 
+
+4. **Ability to toggle between auto-build and manual build settings** Have control over when you want to implement Codewind's auto-build feature.
+
+5. **Project details displayed in a user-friendly panel on your IDE** Use Codewind's Project Overview interface to instantly view the endpoint, container ID, location on disk, build and run status, and ports.
+
+6. **Access application, build and container logs and run commands from within the container without leaving your IDE** Codewind integrates with your IDE's terminal window, 'problems' and 'output' panels for a smooth test and debug experience. Additionally, you can open a shell session *inside* your container with a simple click.
+
+7. **Support for your IDE's debug features**<sup>*</sup>  Full compatiility with the debug functionality of your IDE.
```

Comment:
- Thus far, I don't think I've seen superscripts in our documentation/website. Could we just add the words "where supported" as part of the title/text instead of adding it as a footnote?
- Two empty spaces appear in front of "Full." I'm assuming this double space is a typo?
- Also typo in "compatiility," which I assume should be "compatibility." :)

Reply:
Done